desktop/src/main/java/bisq/desktop/app/BisqApp.java

```diff
@@ -275,7 +275,7 @@ private void addSceneKeyEventHandler(Scene scene, Injector injector) {
                     injector.getInstance(SendAlertMessageWindow.class).show();
                 } else if (Utilities.isAltOrCtrlPressed(KeyCode.F, keyEvent)) {
                     injector.getInstance(FilterWindow.class).show();
-                } else if (Utilities.isAltOrCtrlPressed(KeyCode.UP, keyEvent)) {
```

Comment:
Comments should be updated https://github.com/bisq-network/bisq/blob/3854907c14357680038661c8153095a157efbc5d/core/src/main/java/bisq/core/dao/governance/voteresult/MissingDataRequestService.java#L81

Reply:
Thanks.